src/object.rs

```diff
@@ -120,6 +128,24 @@ impl Object {
     unsafe { v8__Object__CreateDataProperty(self, &*context, &*key, &*value) }
   }
 
+  /// Implements DefineOwnProperty.
+  ///
+  /// In general, CreateDataProperty will be faster, however, does not allow
```

Comment:
Should I s/CreateDataProperty/create_data_property/ here?

Reply:
We have just been copying docstrings directly, so I wouldn't worry about it. 

In the future we hope to automate this somehow - maybe with some sort of smart switch/replace? But I think just having some docs is useful.